wizeline/wize-q-front
# Feat - WIP 263 - Add Pagination Functionality

#### What does this PR do?
- This feature replaces the _Show more..._ button that was previously used with pagination buttons, now displaying pages with 20 questions each.

#### Where should the reviewer start?
View the new functions added in ListQuestions.js used to handle the fetchQuestions function with pagination functionality and see the render for the Pager. Afterwards see the App.css file.

#### How should this be manually tested?
1. Once logged in, the user can use buttons in the bottom of the page to navigate through the question pages.
2. Click the different buttons to navigate to the last, first, or a specific page.
3. If there are more than 5 pages of questions, an ellipsis button is displayed in the pagination.
4. The _next_ and _previous_ buttons are disabled when reaching boundaries.
5. This works the same way in the answered questions tab.

#### What are the relevant tickets?
[Add pagination functionality](https://wizeline.atlassian.net/browse/WIP-263)

#### Screenshots
![image](https://user-images.githubusercontent.com/15947670/27942934-b80147ea-62a0-11e7-8595-286570387a84.png)
Pending questions

![image](https://user-images.githubusercontent.com/15947670/27942950-c466314e-62a0-11e7-87c5-4676e1dac6ab.png)
Answered questions

#### Questions
Would it be convenient to place the pagination on top of the question list as well?
